cmd/vulcan-semgrep/main.go

```diff
@@ -259,7 +215,7 @@ func vuln(result Result, filepath string, vulns map[string]report.Vulnerability)
 		cweID, _ = strconv.Atoi(matches[1])
 	}
 
-	summary = strings.Title(summary)
+	summary = cases.Title(language.English).String(summary)
```

Comment:
Does this change any of the existent summaries? Take into consideration that a change on the summary is a change on the issue and therefore we loose track of the findings.

Reply:
Good catch! It shouldn't but not 100% sure. I reverted the change.